exporter/solarwindsexporter/factory.go

```diff
@@ -45,7 +45,10 @@ func createMetricsExporter(
 		return nil, fmt.Errorf("unexpected config type: %T", cfg)
 	}
 
-	metricsExporter := newExporter(ctx, exporterCfg, settings, metricsExporterType)
+	metricsExporter, err := newExporter(exporterCfg, settings, metricsExporterType)
+	if err != nil {
+		return nil, err // TODO: Wrap.
```

Comment:
just adding explicit comment to make sure the TODO is either resolved or comment removed